hw/top_earlgrey/ip/ast/data/ast.hjson

```diff
@@ -1,6 +1,10 @@
 // Copyright lowRISC contributors.
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
+###############################################################################
+# *Name: ast
+# *Module Description: Analog Sensors Top Registers
+###############################################################################
```

Comment:
just curious, why is a different comment format used here? both work, but we typically just stick to `//` for uniformity...

Reply:
I think we took an example from one of the existing files, but we will fix
This hybrid comment format also appears in the following files:

./hw/dv/data/xcelium/xcelium.hjson
./hw/ip/aes/data/aes.hjson
./hw/ip/alert_handler/data/alert_handler.hjson
./hw/ip/clkmgr/data/clkmgr.hjson
./hw/ip/nmi_gen/data/nmi_gen.hjson
./hw/ip/padctrl/data/padctrl.hjson
./hw/ip/pinmux/data/pinmux.hjson
./hw/ip/rstmgr/data/rstmgr.hjson
./hw/ip/rv_plic/data/rv_plic.hjson
./hw/ip/sensor_ctrl/data/sensor_ctrl.hjson
./hw/top_earlgrey/ip/alert_handler/data/autogen/alert_handler.hjson
./hw/top_earlgrey/ip/clkmgr/data/autogen/clkmgr.hjson
./hw/top_earlgrey/ip/padctrl/data/autogen/padctrl.hjson
./hw/top_earlgrey/ip/pinmux/data/autogen/pinmux.hjson
./hw/top_earlgrey/ip/rv_plic/data/autogen/rv_plic.hjson
./hw/vendor/lowrisc_ibex/dv/uvm/data/xcelium/xcelium.hjson

Reply:
that would be great, thanks!